lib/web_ui/test/engine/platform_dispatcher_test.dart

```diff
@@ -160,6 +162,34 @@ void testMain() {
       expect(ui.PlatformDispatcher.instance.textScaleFactor, findBrowserTextScaleFactor());
     });
   });
+
+  test('ViewConfiguration asserts that both window and view are not provided', () {
+    expect(() {
+      // ignore: deprecated_member_use
+      final EngineFlutterWindow window = EngineFlutterWindow(0, ui.PlatformDispatcher.instance);
+      ui.ViewConfiguration(
+        window: window,
+        view: window,
+      );
+    }, throwsAssertionError);
+  });
+
+  test("A ViewConfiguration's view and window are backed with the same property", () {
+    final ui.FlutterView window = EngineFlutterWindow(0, ui.PlatformDispatcher.instance);
+    final ui.ViewConfiguration viewConfiguration = ui.ViewConfiguration(view: window);
+    // ignore: deprecated_member_use
+    expect(viewConfiguration.window, window);
+    // ignore: deprecated_member_use
+    expect(viewConfiguration.window, viewConfiguration.view);
+  });
+
+  test('Calling copyWith on a ViewConfiguration with both a window and view throws an error', () {
+    final ui.FlutterView window = EngineFlutterWindow(0, ui.PlatformDispatcher.instance);
```

Comment:
Can you add positive test cases for `copyWith`? Test that `copyWith` works with `window` and with `view`.

Reply:
I don't think that there's a way to create another `FlutterView` to copy over the `ViewConfiguration`'s owned view since Flutter is only single-view for the moment. 

Let me know if I'm missing something, but I'm not sure if I can resolve this suggestion.